lib/sgtn-client/api/translation.rb

```diff
@@ -131,9 +130,11 @@ def self.compare_source(translation_bundle, old_source_bundle, source_bundle)
       translation_messages = translation_bundle['messages']
       translation_bundle['messages'] = new_translation_messages = {}
       source_bundle['messages'].each do |key, value|
-        new_translation_messages[key] = if old_source_messages[key] == value
-                                          translation_messages[key] || value
+        translation = translation_messages[key]
```

Comment:
Also, we don't want to repetitively return an empty translation bundle in line 127.  I'd like to cache the fallback information once and use it until it expires.

Reply:
This way, the approach will be consistent: no more post-processing after caching regardless if translation is missing for only one key or for the whole bundle. We also avoid multiple requests for the same missing bundle.